src/test/java/io/github/dsheirer/CommonFixtures.java

```diff
@@ -0,0 +1,23 @@
+package io.github.dsheirer;
+
+import io.github.dsheirer.channel.IChannelDescriptor;
+import io.github.dsheirer.identifier.IdentifierCollection;
+import io.github.dsheirer.identifier.Role;
+import io.github.dsheirer.module.decode.dmr.channel.DMRLogicalChannel;
+import io.github.dsheirer.module.decode.dmr.identifier.DMRTalkgroup;
+
+import java.util.List;
+
+public interface CommonFixtures {
```

Comment:
this will likely evolve into per-entity fixtures. for now it's enough to have a common support interface that can be mixed in to be able to construct basic building blocks with some default values